debug-db/src/main/java/com/amitshekhar/sqlite/DebugSQLiteDB.java

```diff
@@ -3,6 +3,7 @@
 import android.content.ContentValues;
 import android.database.Cursor;
 import android.database.SQLException;
+import android.util.Log;
```

Comment:
Only an (unused) import statement was added in this file. It would be good to remove it again before the merge ;)